spi_engine/Makefile

```diff
@@ -12,13 +12,18 @@ SV_DEPS += ../common/sv/m_axis_sequencer.sv
 SV_DEPS += ../common/sv/s_axis_sequencer.sv
 SV_DEPS += ../common/sv/m_axi_sequencer.sv
 SV_DEPS += ../common/sv/s_axi_sequencer.sv
+SV_DEPS += ../common/sv/spi_vip/adi_spi_vip_pkg.sv
+SV_DEPS += ../common/sv/spi_vip/spi_vip_if.sv
+SV_DEPS += ../common/sv/spi_vip/adi_spi_vip.sv
```

Comment:
Lib deps shouldn't be listed here since they are already component.xml-timestamp-managed
```suggestion
```